tests/examples/test_quantizing_moe.py

```diff
@@ -0,0 +1,75 @@
+import shlex
```

Comment:
Just so that you're aware: https://github.com/vllm-project/llm-compressor/pull/192/files#diff-7311b466a391f25c3d0951bbba59f2b3db9b3391ccc0c5949b22320b1e63a39e